integration/spark/src/test/scala/org/apache/carbondata/view/timeseries/TestCreateMVWithTimeSeries.scala

```diff
@@ -33,17 +34,30 @@ class TestCreateMVWithTimeSeries extends QueryTest with BeforeAndAfterAll {
 
   override def beforeAll(): Unit = {
     CarbonProperties.getInstance().addProperty(CarbonCommonConstants.CARBON_TIMESTAMP_FORMAT, "dd-MM-yyyy")
-    drop()
+    dropTable()
     sql("CREATE TABLE maintable (empname String, designation String, doj Timestamp, workgroupcategory int, workgroupcategoryname String, deptno int, " +
-        "deptname String, projectcode int, projectjoindate Timestamp, projectenddate Timestamp,attendance int, utilization int,salary int) STORED AS carbondata")
+      "deptname String, projectcode int, projectjoindate Timestamp, projectenddate Timestamp,attendance int, utilization int,salary int) STORED AS carbondata")
+    sql(s"""
+         | CREATE INDEX maintable_index_test
+         | ON TABLE maintable (designation)
+         | AS '${classOf[WaitingIndexFactory].getName}'
+    """.stripMargin)
+
     sql(s"""LOAD DATA local inpath '$resourcesPath/data_big.csv' INTO TABLE maintable  OPTIONS
          |('DELIMITER'= ',', 'QUOTECHAR'= '"')""".stripMargin)
+
+    sql("CREATE TABLE temp_maintable (empname String, designation String, doj Timestamp, workgroupcategory int, workgroupcategoryname String, deptno int, " +
```

Comment:
create table again?

Reply:
this is temp_maintable here, this temporary table is used to insert into maintable in order to make "insert into or insert overwrite" operation in the following.